website/src/pages/community/index.tsx

```diff
@@ -86,14 +86,21 @@ export default function Home(): JSX.Element {
                 <li>
                   <a href={useBaseUrl('/tutorial')}>Tutorials</a>: Step-by-step guides to get started.
                 </li>
+                <li>
+                  <a href="https://github.com/podman-desktop/community/blob/main/presentations/README.md">
```

Comment:
```suggestion
                  <a href={useBaseUrl('/community/presentations/README.md')}>
```

Reply:
@jeffmaury This is an external link not a part of this repo, hence had removed `useBaseUrl`

Reply:
This is not an external link it is referencing a page that is part of the podman desktop website and useBaseUrl is used for that so that it works in any branch

Reply:
so the link for presentations and demos is not a part of the podman-desktop.io but these readmes reside on the [community](https://github.com/podman-desktop/community) repo. If i have to use `<a href={useBaseUrl('/community/presentations/README.md')}>` then it resolves as` http://localhost:3000/community/presentations/README.md` which is a page not found

Reply:
Yes my bad I was missing a layer in the repository